_sass/typography.scss

```diff
@@ -141,6 +141,10 @@ p {
 	}
 }
 
+.justify-text {
```

Comment:
Nop, I asked you yesterday why there was no SASS compilation in the Gruntfile. The answer is that Jekyll is compiling SASS when starting. That's awesome, isn't it ?